src/ontology/mondo-edit.obo

```diff
@@ -296798,7 +296762,6 @@ xref: SCTID:722207000 {source="MONDO:equivalentTo"}
 xref: UMLS:C2675184 {source="MEDGEN:436369", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 is_a: MONDO:0001684 {source="MESH:C567195"} ! exocrine pancreatic insufficiency
 is_a: MONDO:0002356 {source="MONDO:Redundant", source="Orphanet:199337"} ! pancreas disorder
-is_a: MONDO:0009068 ! cytochrome-c oxidase deficiency disease
```

Comment:
@nicolevasilevsky I don't think this x-ref should have been removed. It should have been replaced by the "replaced by" ID.

Reply:
fixed